Airports/EGHQ/Ownership.txt

```diff
@@ -1,5 +1,5 @@
 SECTOR:HQTWR:0:2000
-OWNER:HQT:HQR:HQA:LW:LWN:LWC:LWS:L
+OWNER:HQT:HQR:HQA:LWL:LWX:LW:LWN:LWC:LWS:L
```

Comment:
```suggestion
OWNER:HQT:HQR:HQA:LWH:LWX:LW:LWN:LWC:LWS:L
```
Should be sector 6 not 9?

Reply:
Think Sector 9 makes most sense?

![image](https://user-images.githubusercontent.com/12017284/91635090-6b9a0b80-e9ed-11ea-9b26-bfa01ffd53e8.png)

Reply:
Yep, agreed original is correct.